docs/serving/using-auto-tls.md

```diff
@@ -40,8 +40,8 @@ You must meet the following prerequisites to enable auto TLS:
 - The following must be installed on your Knative cluter:
   - [Knative Serving](../install/).
   - [Istio with SDS, version 1.3 or higher](../install/installing-istio.md#installing-istio-with-SDS-to-secure-the-ingress-gateway),
-    [Contour, version 1.1 or higher](../install/Knative-with-Contour.md),
-    or [Gloo, version 0.18.16 or higher](../install/Knative-with-Gloo.md).
+    [Contour, version 1.1 or higher](https://github.com/knative/docs/blob/master/docs/install/knative-with-any-k8s.md#installing-the-serving-component),
```

Comment:
I think here you want to link to `../install/any-kubernetes-cluster.md#installing-the-serving-component`.

Reply:
Yes. Thanks. I have fixed it.